test/java/com/odysseusinc/arachne/portal/front/StudyManagerTest.java

```diff
@@ -1,61 +1,699 @@
 import org.openqa.selenium.support.ui.WebDriverWait;
 
-public class StudyManagerTest extends BaseTest {
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class StudyManagerTest extends BaseDataCatalogTest {
 
-    private static final String MODAL_TITLE_CREATE_STUDY = "Create study";
-    private static final String PLACEHOLDER_STUDY_TITLE = "Name of study";
-    private static final String PLACEHOLDER_STUDY_TYPE = "Type";
+    public static final String MODAL_TITLE_CREATE_STUDY = "Create study";
+    private static final String MODAL_TITLE_CREATE_ANALYSIS = "Create analysis";
+    private static final String MODAL_TITLE_UPDATE_STUDY_TITLE = "Update study title";
 
-    @Test
-    public void testCreateStudy() throws Exception {
+    public static final String PLACEHOLDER_STUDY_NAME = "Name of study";
+    public static final String PLACEHOLDER_STUDY_TYPE = "Type";
+    public static final String PLACEHOLDER_STUDY_TITLE = "Title of study";
+    public static final String PLACEHOLDER_ANALYSIS_TITLE = "Title";
 
-        By addBtn = ByBuilder.buttonAddIco();
+    private static final String BEFORE_UPDATING_STUDY_NAME = "TEST Stu";
+    private static final String STUDY_NAME = "TEST Study";
 
-        String studyName = "New study " + new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(new Date());
-        String studyType = "Clinical Trial Design";
+    private static final String NAME_DS = "TestNode: Test Data Source 2";
+    private static final String NAME_FOR_DELETED_STUDY = "Study For Deleting";
+
+    private static final BaseStudyTest.StudyData STUDY_DATA =
+            new BaseStudyTest.StudyData(BEFORE_UPDATING_STUDY_NAME, "Clinical Trial Design", "Initiate");
+
+    private static final BaseStudyTest.StudyData STUDY_FOR_DELETING_DATA =
+            new BaseStudyTest.StudyData(NAME_FOR_DELETED_STUDY, "Clinical Trial Design", "Initiate");
+
+    private static File file;
+
+    @BeforeClass
+    public static void beforeStudyManagerTests() throws IOException {
+
+        updateName(ADMIN_LOGIN, ADMIN_PASSWORD, "4");
+        updateName("mr_lead_investigator@example.com", "password", "1");
+        updateName("mr_data_set_owner@example.com", "password", "2");
+        updateName("mr_collaborator@example.com", "password", "3");
+
+        file = new File("text.txt");
+        file.createNewFile();
+    }
+
+    @AfterClass
+    public static void afterStudyManagerTests() throws IOException {
+
+        file.delete();
+    }
+
+    @After
+    public void afterStudyManagerTest() throws IOException {
+
+        logout();
+    }
+
+    @Test
+    public void test01CreateStudy() throws Exception {
 
         loginPortal(ADMIN_LOGIN, ADMIN_PASSWORD);
-        waitForPageLoad(driver, addBtn);
+        createStudy(STUDY_DATA);
+    }
+
+    @Test
+    public void test02UpdateStudyTitle() throws Exception {
+
+        loginAndOpenStudy(BEFORE_UPDATING_STUDY_NAME);
+
+        By editBtn = ByBuilder.toolbarEditIco("edit");
```

Comment:
Why you name method "toolbarEditIco" and then, again, explicitely pass "edit" param?

Reply:
"EditIco" is from class name, but text may be different in common case